cfme/tests/test_appliance.py

```diff
@@ -435,3 +435,24 @@ def codename_in_stdout():
         r = appliance.ssh_client.run_command(
             r'journalctl -u evmserverd -c "{}" | egrep -i "codename: \w+$"'.format(cursor))
         return r.success
+
+
+def test_fixauth_dryrun_has_feedback(temp_appliance_preconfig):
+    """
+    Check whether the fixauth says it is running in dry mode
+
+    Polarion
+        assignee: jhenner
+        casecomponent: Appliance
+        initialEstimate: 1/60h
+
+    Testing BZ 1577303
```

Comment:
I missed this... @jaryn we started Bugzilla collection consistency under a common tag like 

https://github.com/ManageIQ/integration_tests/blob/c5cba3736dea0110a5d01a6d345d61214f4f6102/cfme/tests/automate/test_domain.py#L273

Also if you are automating you and add `automate` marker.